<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>992</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #cecbc8;
color: #614e41;
font-size: 22px;
border-radius: 10px;
padding: 20px 10px; /* top &amp; bottom padding bigger */
font-weight: bold;
qproperty-alignment: AlignCenter;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="containerframe">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>381</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#containerframe {
    background-color: #28221e;       
    border-radius: 15px;            
    border: 2px solid #614e41;       
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>341</width>
       <height>445</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="labelFareDisplay">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="217">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>206</red>
              <green>203</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>40</red>
              <green>34</green>
              <blue>30</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>40</red>
              <green>34</green>
              <blue>30</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>206</red>
              <green>203</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>206</red>
              <green>203</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>40</red>
              <green>34</green>
              <blue>30</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="217">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>206</red>
              <green>203</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>40</red>
              <green>34</green>
              <blue>30</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>40</red>
              <green>34</green>
              <blue>30</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>206</red>
              <green>203</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>206</red>
              <green>203</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>40</red>
              <green>34</green>
              <blue>30</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>40</red>
              <green>34</green>
              <blue>30</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>206</red>
              <green>203</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>40</red>
              <green>34</green>
              <blue>30</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>40</red>
              <green>34</green>
              <blue>30</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>206</red>
              <green>203</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>206</red>
              <green>203</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>40</red>
              <green>34</green>
              <blue>30</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#labelFareDisplay {
    background-color: #cecbc8;
    color: #28221e;
    border: 2px solid #614e41;
    border-radius: 10px;
    padding: 8px;
    font-size: 18px;
    font-weight: bold;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="text">
         <string>Fare: Rs. 00.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEnterBus">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #614e41;
    color: #ffffff;
    border: none;
    border-radius: 10px;
    padding: 10px 20px;
    font-weight: bold;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #7f6951;
}

QPushButton:pressed {
    background-color: #4f3d33;
}
</string>
        </property>
        <property name="text">
         <string>Enter Bus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExitBus">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #614e41;
    color: #ffffff;
    border: none;
    border-radius: 10px;
    padding: 10px 20px;
    font-weight: bold;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #7f6951;
}

QPushButton:pressed {
    background-color: #4f3d33;
}
</string>
        </property>
        <property name="text">
         <string>Exit Bus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRechargeCard">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #614e41;
    color: #ffffff;
    border: none;
    border-radius: 10px;
    padding: 10px 20px;
    font-weight: bold;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #7f6951;
}

QPushButton:pressed {
    background-color: #4f3d33;
}
</string>
        </property>
        <property name="text">
         <string>Recharge Card</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdminPanel">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #614e41;
    color: #ffffff;
    border: none;
    border-radius: 10px;
    padding: 10px 20px;
    font-weight: bold;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #7f6951;
}

QPushButton:pressed {
    background-color: #4f3d33;
}
</string>
        </property>
        <property name="text">
         <string>Admin Panel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #614e41;
    color: #ffffff;
    border: none;
    border-radius: 10px;
    padding: 10px 20px;
    font-weight: bold;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #7f6951;
}

QPushButton:pressed {
    background-color: #4f3d33;
}
</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="Header">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>441</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #28221e;
color: #cecbc8;
font-size: 24px;
border-radius: 10px;
padding: 10px;
font-weight: bold;
qproperty-alignment: AlignCenter;
</string>
    </property>
    <property name="text">
     <string>Smart Bus Fare System</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelUpcomingCheckpoint">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>640</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #614e41;
color: #cecbc8;
font-size: 24px;
border-radius: 10px;
padding: 10px;
font-weight: bold;
qproperty-alignment: AlignCenter;
</string>
    </property>
    <property name="text">
     <string>Smart Bus Fare System</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDirection">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>720</y>
      <width>411</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #cecbc8;
color: #28221e;
border: 2px solid #614e41;
border-radius: 10px;
padding: 10px;
font-size: 20px;
font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>Direction</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1222</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
